admin/taxonomy/class-taxonomy-metabox.php

```diff
@@ -205,10 +206,7 @@ private function create_collapsible( $name, $network, $icon, $label ) {
 		$tab_settings = new WPSEO_Metabox_Collapsible(
 			$name,
 			$this->social_admin->get_premium_notice( $network ) . $this->taxonomy_tab_content->html( $meta_fields ),
-			$label,
-			array(
-				'single' => $this->has_single_social_tab(),
```

Comment:
As far as I can see, `has_single_social_tab` isn't used anywhere in the code anymore. Please remove/deprecate the function.